scripts/Interface/ui/reflectometer/refl_gui.py

```diff
@@ -1006,19 +1001,24 @@ def _do_run(self, runno, row, which):
         ws = ConvertToWavelength.to_workspace(loadedRun, ws_prefix="")
 
         if self.__alg_use:
+            if self.tableMain.item(row, self.scale_col).text():
+                factor=float(self.tableMain.item(row, self.scale_col).text())
+            else:
+                factor=1.0
+            if self.tableMain.item(row, 15).text():
+                momentum_transfer_step=float(self.tableMain.item(row, 15).text())
```

Comment:
Shouldn't we read `qmin` and `qmax` as well? Otherwise `ReflectometryReductionOne` will always set `MomentumTransferMin` and `MomentumTransferMax` to the default values I think.